### PR TITLE
add base URL to API section of Docs

### PR DESCRIPTION
**Pull request by JamesLDeveloper**
image

Have added the base URL for API and base URL for public endpoints to Header in API section of docs

### CODE REVIEW

**Comment by RichardHill** [comment]
Please can you make that url an actual link so someone can just click it ?

**Comment by JamesLDeveloper** [comment]
image

Have modified the default page construction by adding DocPage/index.js to include a div only for the api section which includes clickable links for the API

File: src/theme/DocPage/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  useEffect(() => {
+    if (isApiCategory) {
+      const articleElement = document.querySelector('article.margin-top--lg');
+      if (articleElement && !document.querySelector('.api-info')) {
+        const apiInfoDiv = document.createElement('div');
+        apiInfoDiv.className = 'api-info';
+        apiInfoDiv.innerHTML = `
+          <p>
+            The base URL for the API is <a href="https://api.greencloud.dev/v1">https://api.greencloud.dev/v1</a>
```
Comment (tomaswarynyca) [inline src/theme/DocPage/index.js:17]:
```suggestion
            The base URL for the API is <a href="https://api.greencloud.dev/v1/">https://api.greencloud.dev/v1/</a>
```

File: src/theme/DocPage/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            </p>
+            <p>
+            The base URL to access your created public endpoints is <a href="https://api.greencloud.dev/gc">https://api.greencloud.dev/gc</a>
```
Comment (tomaswarynyca) [inline src/theme/DocPage/index.js:20]:
```suggestion
            The base URL to access your created public endpoints is <a href="https://api.greencloud.dev/gc/">https://api.greencloud.dev/gc/</a>
```

**Review by tomaswarynyca — APPROVED** [review]

**Pull request merged**